lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = []
+    self.reset!
+  end
+
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def display
+    puts " "
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    puts " "
+  end
+
+  def position(input)
+    self.cells[input.to_i - 1]
+  end
+
+  def full?
+    self.cells.all? {|cell| cell != " "}
+  end
+
+  def turn_count
+    count = 0
+    self.cells.each {|cell| count += 1 if cell != " "}
+    count
+  end
+
+  def taken?(position)
+    self.cells[position.to_i - 1] == " " ? false : true
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. Additionally, you can reverse (or negate) the boolean with the `!` (bang) operator.

`self.cells[position.to_i - 1] != " "`